setup.py

```diff
@@ -32,5 +32,7 @@
                      'numpy',
                      'sklearn',
                      'pycocotools',
-                     'pycocoevalcap'
+                     'pycocoevalcap',
```

Comment:
Could you help update requirements.txt as well? Thanks!

Reply:
added!